optaplanner-core/src/main/java/org/optaplanner/core/impl/score/buildin/bendable/BendableScoreHolderImpl.java

```diff
@@ -36,9 +35,9 @@
  */
 public final class BendableScoreHolderImpl extends AbstractScoreHolder<BendableScore> implements BendableScoreHolder {
```

Comment:
The changes to all score holders all do the same thing. It's just a whole bunch of code duplication.
If you've reviewed the changes to one score holder, you've reviewed them all.